doc/man7/pbkdf2.pod

```diff
@@ -0,0 +1,129 @@
+=pod
```

Comment:
This all depends on what you actually want to document and how that documentation should be reached.  man7 pages should be general descriptions on a topic, so a general description of PBKDF2 in OpenSSL would be fitting for man7.  These would typically not have a SYNOPSIS section, or a very short one, mostly referring to the header associated with the stuff described (see for example bio.pod).

However, there's also the documentation of specific functions (or macros as the case may be), and they should rather be in man3, with a collection of the function names in the NAME section and a fleshed out SYNOPSIS section.

With that in mind, I agree with @mattcaswell regarding this page, it looks much more like a man3 page than a man7 one.